src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -302,14 +302,13 @@ class GlobalSelectionHeader extends React.Component {
 
   getProjects = () => {
     const {isSuperuser} = ConfigStore.get('user');
-    const {projects, organization} = this.props;
-    const unfilteredProjects = projects || organization.projects;
```

Comment:
@Chrissy I seem to recall this being here for a reason, do you remember why?

Reply:
It's probably a relic of before the component was wrapped in `withProjects`

Reply:
Here is the commit that introduced the behavior https://github.com/getsentry/sentry/commit/109ba9c1dd68d19a5d80e6800e9abc67c3a0643c

Reply:
yeah afaik this is a relic. there is similar looking code in `projectSelector` that _is_ important because it can either receive projects as a prop or get it from the state, but this is not that.